src/Context/CategoryContext.js

```diff
@@ -0,0 +1,25 @@
+import react from "react";
+import { createContext } from "react";
+import { useContext } from "react";
+import { useState } from "react";
+import { useEffect } from "react";
```

Comment:
You may import them together (in a single line of code) for better readability and clean code